content/knowledge-base/migrating-from-bitrise.md

```diff
@@ -0,0 +1,202 @@
+---
+description: How to migrate from Bitrise to Codemagic
+title: Migrating from Bitrise
+weight: 13
+---
+
+Codemagic makes use of [`codemagic.yaml`](../yaml/yaml-getting-started/) for configuring your workflow. Codemagic supports any Git-based cloud or self-hosted repository. Thus, there is no need to migrate your code; simply add a `codemagic.yaml` file to the repo. In Codemagic, there is also a [Flutter workflow editor](../flutter-configuration/flutter-projects/) for Flutter applications, which simplifies the setup but removes some flexibility.
+
+## Migrating iOS builds with codemagic.yaml
+
+If you have already set up your iOS application on Bitrise, migrating to Codemagic is straightforward.
+
+### Code signing
+
+Like Bitrise, Codemagic lets users configure either a manual or an automatic option for handling iOS code signing profiles and certificates.
+
+#### Manual
+
+Codemagic requires you to add all of your signing files (profiles, certificates, etc.) as environment variables with the manual option. These are the same files uploaded on Bitrise under the applications `Code Signing` tab.
+
+If you have already set up manual code signing on Bitrise, setting it up with Codemagic is relatively simple. You can download the necessary provisioning profiles and certificates from Bitrise's `Code Signing` tab. To add them as environment variables in Codemagic, navigate to your application and click the `Environment variables` tab. Encode the contents of your files with base64 and paste the values into Codemagic; make sure to check `Secure` when providing sensitive information. Along with creating these variables, create a new variable group, e.g., `ios_code_signing`, which you can later reference in your `codemagic.yaml`. For convenience, we recommend naming your variables as `FCI_CERTIFICATE`, `FCI_CERTIFICATE_PASSWORD`, and `FCI_PROVISIONING_PROFILE`. However, you can call the variables differently if you reference them correctly in the scripts section. In the case of multiple provisioning profiles, the recommended naming convention is `FCI_PROVISIONING_PROFILE_1`, `FCI_PROVISIONING_PROFILE_2` etc.
```

Comment:
This paragraph is too long. I like how you added the required variables as a code block for Android code signing. Let's shorten and say it's required to save them as secure env vars in the UI and you only have to import the group in yaml.